Tests/Provider/VimeoProviderTest.php

```diff
@@ -147,6 +147,30 @@ public function testTransformWithUrl()
         $this->assertEquals('012341231', $media->getProviderReference(), '::getProviderReference() is set');
     }
 
+	public function testTransformWithPlayerUrl()
```

Comment:
@OskarStark thank you for pointing out, and sorry for this, I used a text editor that time, not IDE, so it messed it up with a tab
@Soullivaneuh I've restricted the regexp as you said